analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentTask.java

```diff
@@ -118,17 +123,27 @@ private void writeMarkerFile(File rootDir) throws IOException {
 
             if (configuration.isCanBeResolved()) {
 
+                final Set<ProjectRef> allProjectDependencies = configuration.getAllDependencies()
```

Comment:
Should this not be getDependencies() rather than getAllDependencies?

Reply:
I tried `getDependencies()` and was getting 0 dependencies back for the new test case.

Reply:
Ok (discussed offline). Perhaps add a comment for future reference.

Reply:
:+1:, comment added